webpack/localization-plugin/src/LocalizationPlugin.ts

```diff
@@ -439,15 +499,15 @@ export class LocalizationPlugin implements Webpack.Plugin {
     return this._stringPlaceholderMap.get(serialNumber);
   }
 
-  private _addLocFile(localeName: string, locFilePath: string, locFileData: ILocaleFileData): void {
+  private _addLocFile(localeName: string, localizedFilePath: string, localizedFileData: ILocaleFileData): void {
     const filesMap: Map<string, Map<string, string>> = this._resolvedLocalizedStrings.get(localeName)!;
 
     const stringsMap: Map<string, string> = new Map<string, string>();
-    filesMap.set(locFilePath, stringsMap);
+    filesMap.set(localizedFilePath, stringsMap);
 
-    for (const stringName in locFileData) {
-      if (locFileData.hasOwnProperty(stringName)) {
-        const stringKey: string = `${locFilePath}?${stringName}`;
+    for (const stringName in localizedFileData) {
```

Comment:
We can write this a bit more ergonomically using `Object.entries` which makes conversion to/from `Map` easy.
```ts
for (const [stringName, stringValue] of Object.entries(locFileData))
```